--- origsrc/luaexpat-1.1/makefile	2006-06-08 15:41:48.000000000 -0500
+++ src/luaexpat-1.1/makefile	2008-02-13 12:17:12.953125000 -0600
@@ -12,7 +12,7 @@
 lib: src/$(LIBNAME)
 
 src/$(LIBNAME) : $(OBJS)
-	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) -o src/$(LIBNAME) $(LIB_OPTION) $(OBJS) -lexpat
+	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) -o src/$(LIBNAME) $(LIB_OPTION) $(OBJS) -lexpat $(LIBS)
 
 $(COMPAT_DIR)/compat-5.1.o: $(COMPAT_DIR)/compat-5.1.c
 	$(CC) -c $(CFLAGS) -o $@ $(COMPAT_DIR)/compat-5.1.c
@@ -20,7 +20,6 @@
 install:
 	mkdir -p $(LUA_LIBDIR)
 	cp src/$(LIBNAME) $(LUA_LIBDIR)
-	cd $(LUA_LIBDIR); ln -f -s $(LIBNAME) $T.so
 	mkdir -p $(LUA_DIR)/$T
 	cp src/$T/lom.lua $(LUA_DIR)/$T
 
