--- origsrc/luaexpat-1.3.3/makefile	2018-11-27 20:31:28.000000000 +0900
+++ src/luaexpat-1.3.3/makefile
@@ -45,7 +45,7 @@ OBJS= src/lxplib.o
 lib: src/$(LIBNAME)
 
 src/$(LIBNAME) : $(OBJS)
-	$(CC) $(CFLAGS) -o src/$(LIBNAME) $(LIB_OPTION) $(OBJS) -lexpat
+	$(CC) $(CFLAGS) -o src/$(LIBNAME) $(LIB_OPTION) $(OBJS) $(LIBS)
 
 install:
 	mkdir -p $(LUA_LIBDIR)
@@ -50,7 +50,6 @@ src/$(LIBNAME) : $(OBJS)
 install:
 	mkdir -p $(LUA_LIBDIR)
 	cp src/$(LIBNAME) $(LUA_LIBDIR)
-	cd $(LUA_LIBDIR); ln -f -s $(LIBNAME) $T.so
 	mkdir -p $(LUA_DIR)/$T
 	cp src/$T/lom.lua $(LUA_DIR)/$T
 
